kernel/src/main/java/org/kframework/parser/json/JsonParser.java

```diff
@@ -317,11 +318,19 @@ public static K parse(byte[] data) {
     }
 
     public static K parse(String data) {
+        return parse(data, Module.apply("dummy", immutable(Sets.newHashSet())));
+    }
```

Comment:
This is just so we can feed in an optional `Module` parameter, or can skip it? Let's just make it required everywhere.

Reply:
Or is this type-signature used in several places? I guess it makes sense then. Is this a normal way to do it? Should we call it the `#JSON-PARSING-MODULE` or something?